### PR TITLE
Translations for sbpp checker

### PR DESCRIPTION
**Pull request by sneak-it**
## Description

I have updated sbpp_checker to utilize translations, defined a prefix as with sbpp_main & sbpp_comms, and tweaked a small bit of the language (english) to be a bit more grammatically correct. I have also updated sbpp_sleuth to utilize a prefix, to be uniform with all other plugins, just like above.

## Motivation and Context

Don't think this fixes any issues or requests, but provides increased multilingual support.

## How Has This Been Tested?

I have tested these changes in depth on both a private and populated public server, however additional testing certainly would not hurt!

## Screenshots (if appropriate):
Screenshot

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

This will require an additional translation file for the checker. Upon next release servers will require this file, so not really breaking.

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by CrazyHackGUT** [comment]
You don't fully utilize translations.
In `PrintToBanAdmins()` you should move `VFormat()` in `for` block, and before calling `VFormat()`, call `SetGlobalTransTarget()`.
This causes a bug when checker is utilize translations, but prints for all clients message on one language.

**Review by CrazyHackGUT — CHANGES_REQUESTED** [review]
https://github.com/sbpp/sourcebans-pp/pull/525#issuecomment-430872055

**Comment by sneak-it** [comment]
Good catch! This should be fixed, thank you.

File: game/addons/sourcemod/scripting/sbpp_checker.sp [hunk 1]
```diff
@@ -477,10 +476,12 @@ void PrintListResponse(int userid, int client, const char[] format, any ...)
 void PrintToBanAdmins(const char[] format, any ...)
 {
 	char msg[128];
-	VFormat(msg, sizeof(msg), format, 2);
-
+	
 	for (int i = 1; i <= MaxClients; i++)
 	{
+		SetGlobalTransTarget(i);
+		VFormat(msg, sizeof(msg), format, 2);
```
Comment (CrazyHackGUT) [inline game/addons/sourcemod/scripting/sbpp_checker.sp]: This two functions should be called after checking, in game client or not.

**Comment by sneak-it** [comment]
Fixed, along with a mistake I made in sleuth that went unnoticed.

**Comment by sneak-it** [comment]
Just saw this commit: https://github.com/sbpp/sourcebans-pp/commit/42e3206b910f3d4af6a68299f6e4abda488d01e7

@Groruk should this translation file be renamed to sbpp_checker.phrases.txt ?

**Comment by Groruk** [comment]
> 
> 
> Just saw this commit: [42e3206](https://github.com/sbpp/sourcebans-pp/commit/42e3206b910f3d4af6a68299f6e4abda488d01e7)
> 
> @Groruk should this translation file be renamed to sbpp_checker.phrases.txt ?

yes, that would be nice. (just to be inline with plugin names)

**Comment by sneak-it** [comment]
Done!

**Review by CrazyHackGUT — APPROVED** [review]

**Review by rumblefrog — APPROVED** [review]

**Pull request merged**